tests/ashley/test_forum_search.py

```diff
@@ -445,3 +452,70 @@ def test_forum_search_empty_public_username(self):
 
     def tearDown(self):
         haystack.connections["default"].get_backend().clear()
+
+    def test_forum_search_several_forums_lti_context_search(self):
+        """Creates forum in different lti_context, make sure user can't
+        search into it, even if he has access to this forum."""
+        user = UserFactory()
+
+        lti_context = LTIContextFactory(lti_consumer=user.lti_consumer)
+        lti_context2 = LTIContextFactory(lti_consumer=user.lti_consumer)
+        forum = ForumFactory()
+        forum2 = ForumFactory()
+        forum3 = ForumFactory()
+        forum.lti_contexts.add(lti_context)
+        forum2.lti_contexts.add(lti_context)
+        forum3.lti_contexts.add(lti_context2)
```

Comment:
May be easier to follow if you name your contexts : `context11`, `context12` and `context2`?

Reply:
oups sorry, don't really get this feedback, what's the difference between context2 and context12 ? In all the other tests of this project, the norm was more to use the variable name lti_context rather than context11.

Reply:
nevermind this comment it's a small matter. It's just that naming variable 1, 2, 3 is hard to follow. Giving them meaningful names is easier. The idea was to convey the name the fact that one context is linked to forum1 and others to forum2.